src/services/types.ts

```diff
@@ -0,0 +1,21 @@
+import { Json, u32 } from "@polkadot/types";
+import React from "react";
+
+export interface SchemaDetails {
+  key: React.Key;
```

Comment:
I don't think you need this `key` field

Reply:
If I don't use the key, I get: 
**react.development.js:209 Warning: Each child in a list should have a unique "key" prop.**

Reply:
Ah I see. I guess that works. You could also just use [Table](https://ant.design/components/table/#API) `rowKey` to then map it to `schema_id`. Either way is fine.